src/main/java/hillel/spring/service/HelloService.java

```diff
@@ -0,0 +1,48 @@
+package hillel.spring.service;
+
+import hillel.spring.database.model.HelloModel;
+import hillel.spring.repository.HelloRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Random;
+
+@Service
+public class HelloService {
+
+    public HelloRepository getHelloRepository() {
```

Comment:
Service shouldn't publish repository. Service is the only class having link to repository. If some other components need access to this repo, they should pass their request through this service.